src/ansys/fluent/core/solver/flobject.py

```diff
@@ -2022,6 +2048,7 @@ def get_root(
         cls, _ = get_cls("", obj_info, version=version)
     root = cls()
     root.set_flproxy(flproxy)
+    root._set_interrupt(interrupt)
```

Comment:
If we set an abstract interrupt handler then that handler can also encapsulate **any** logic therein. That means that the client is not forced to operate in terms of a static list of interruptible command paths. That's the Fluent logic, but it does automatically generalise.

Reply:
Done, Thank you.

Reply:
Yes, this is what we discussed. The only tweak would be
```
self._root(self)._on_interrupt(self)
```
in the `except` block, which is more accurate IMHO.